cmdx.py

```diff
@@ -14,7 +14,8 @@
 
 from maya import cmds
 from maya.api import OpenMaya as om, OpenMayaAnim as oma, OpenMayaUI as omui
-from maya import OpenMaya as om1, OpenMayaMPx as ompx1, OpenMayaUI as omui1
+from maya import OpenMaya as om1, OpenMayaMPx as ompx1, OpenMayaUI as omui1, \
```

Comment:
Death to backslashes.

```py
from maya import (
    OpenMaya as om1,
    OpenMayaMPx as ompx1,
    OpenMayaUI as omui1,
    OpenMayaAnim as oma1
)

Reply:
yeah alright! i never questionned that as that's something pycharm would wrap automatically. but i didn't liked it so i'm happy to know that these "\" are ugly

Reply:
Death to PyCharm too. ;)